src/Mesh/babylon.subMesh.ts

```diff
@@ -89,7 +89,7 @@
         public getMaterial(): Material {
             var rootMaterial = this._renderingMesh.material;
 
-            if (rootMaterial && rootMaterial instanceof MultiMaterial) {
+            if (rootMaterial && rootMaterial.getClassName() === "MultiMaterial") {

```

Comment:
Had to cast to MultiMaterial to get getSubMaterial existence test to work, but casting is just typescript